spacemeshos/smapp
# Add export as CSV feature on Transaction Log screen

It closes #1434 

It was also requested a few times by some community members. It should help them with tax reporting :)

#### How to test
1. Run Smapp: better to take some accounts with a large amount of transactions and rewards, you can use my hacky view-only-wallet (ask me if you need it).
2. Open Transaction Log
3. Choose any Filter except "All"
4. Click on the Export CSV icon
5. Check out contents
